metals/src/main/scala/scala/meta/internal/metals/JavaInteractiveSemanticdb.scala

```diff
@@ -227,38 +227,38 @@ case class JdkVersion(
 
 object JdkVersion {
 
-  def getJavaVersionFromJavaHome(
-      javaHome: AbsolutePath
+  def maybeJdkVersionFromJavaHome(
+      maybeJavaHome: Option[AbsolutePath]
   ): Option[JdkVersion] = {
-
-    def fromReleaseFile: Option[JdkVersion] = {
```

Comment:
This should not be removed. We should only use `quickRunWithShortStringResult` as a last resort.

Reply:
I remember on the linked issue, you concluded the matter in a different way. Could you please take a look [here](https://github.com/scalameta/metals/issues/3536#issuecomment-1110088096) to see what best can be done about it?

Reply:
Ok, no problem.